modules/network/network-managers/security-admin-configurations/main.bicep

```diff
@@ -1,3 +1,7 @@
+metadata name = 'Network Manager Security Admin Configurations'
+metadata description = '''This module deploys an Network Manager Security Admin Configuration.
```

Comment:
OK seems to be the case for all network-managers module metadata descriptions

Reply:
fixed